{{ cookiecutter.repo_name }}/data/.gitignore

```diff
@@ -0,0 +1,13 @@
+# Ignore all files in all sub-directories by default
+# but not the sub-directories themselves
+# and not files in this directory
+**/*
+!**/
+!/*
```

Comment:
Git only tracks files and not directories, so there's no point in trying to have an include rule for directories.